data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,435 @@
+		security 0.2
+
+
+mission "Hai Building Mountaintop [2] Recruits"
+	name "Encourage recruits for Mountaintop"
+	description "Now that the station is going ahead, the Hai need to recruit some people to play the part. Someone needs to sell it to them though."
+	landing
+	source "Hai-home"
+	stopover "Makerplace"
+	stopover "Darkmetal"
+	stopover "Giverstone"
+	stopover "Stonebreak"
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Icelake"
+	stopover "Newhome"
+	stopover "Frostmark"
+	stopover "Heartvalley"
+	stopover "Allhome"
+	stopover "Dustmaker"
+	stopover "Greenbloom"
+	stopover "Skyfarm"
+	stopover "Farwater"
+	stopover "Greenwater"
+	destination "Mountaintop"
+	clearance
+	passengers 1
+	blocked `Terry contacts you and says, "Captain, you're going to need a bunk free for me. Do you have space?" You do not, so you should complete or abort another mission to make room for her, then return here.`
+	on visit
+		dialog `You have arrived at the Mountaintop construction site but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [1b] Return: done"
+	on offer
+		conversation
+			`At Hai-home you are directed to a now familiar government hangar.`
+			`	Sayari, Xilin, and Alondo all send you greeting pings, but are busy at the time of your landing. To your surprise though, Terry is banging on your airlock before too long.`
+			`	"Captain <last>, or should I call you <first>? Anyway, doesn't matter. How would you like to make some money?"`
+			`	The smile she gives you is something like you imagine a large cat giving the tasty morsel it just identified as its food.`
+			`	"Of course you do, everyone likes to make money," she says.`
+			`	"The Hai have put out the call for volunteers to staff the new Mountaintop station when it's finished. While you've been off watching early construction works, we've been gauging public interest and well... it's not going to work. In order to sell this they're going to need at least four thousand volunteers by the end of the week to get a good handle on what they can expect to need in terms of facilities, amenities, supplies, and various livability requirements. Nearly twelve thousand by the time the station is actually built. Our current estimates suggest they'll reach a quarter of that."`
+			`	As she's talking Terry has been laying out charts and graphs, and using a handheld Hai holographic device that she seems to have picked up somewhere to show animated diagrams.`
+			choice
+				`	"How do you know all this?"`
+			`	Terry raises an eyebrow at you, "You remember we have a supercomputer that could predict the economic situation across the whole of human habited space up to fifty years in the future to within 1% accuracy right?"`
+			`	When she puts it that way, it does suddenly feel like a bit of a stupid question.`
```

Comment:
Just a thought, the pc could answer back with 'and how well did that work out for you?' if you want some branching dialogue here.